cmd/server-main.go

```diff
@@ -496,7 +494,16 @@ func serverMain(ctx *cli.Context) {
 		getCert = globalTLSCerts.GetCertificate
 	}
 
-	httpServer := xhttp.NewServer([]string{globalMinioAddr}, criticalErrorHandler{corsHandler(handler)}, getCert)
+	httpServer := xhttp.NewServer([]string{
+		globalMinioAddr,
+		globalMinioAddr,
+		globalMinioAddr,
+		globalMinioAddr,
+		globalMinioAddr,
+		globalMinioAddr,
+		globalMinioAddr,
+		globalMinioAddr,
```

Comment:
What is happening here?

Reply:
This i will revert, I was experimenting having multiple listeners on same port using reusePort feature. 

To allow for multicore performance changes, still experimenting with this.